vitest.config.ts

```diff
@@ -2,9 +2,18 @@ import path from 'path'
 import { defineConfig } from 'vitest/config'
 import vue from '@vitejs/plugin-vue'
 import jsx from '@vitejs/plugin-vue-jsx'
+import Components from 'unplugin-vue-components/vite'
 
 export default defineConfig({
-  plugins: [vue(), jsx()],
+  plugins: [
+    vue(),
+    jsx(),
+    Components({
```

Comment:
same here: a short comment explaining why we have this plugin would be great

Reply:
Is it okay like this?
![image](https://user-images.githubusercontent.com/82131679/192142133-d41c8573-8bc1-476d-be90-98438587a5d4.png)

Reply:
👍